instance/tests/models/test_openedx_instance.py

```diff
@@ -240,6 +240,36 @@ def test_newrelic_configuration(self, mocks, mock_provision):
         self.assertEqual(configuration_vars['COMMON_DEPLOYMENT'], instance.internal_lms_domain)
         self.assertEqual(configuration_vars['NEWRELIC_LICENSE_KEY'], 'newrelic-key')
 
+    @override_settings(
+        INSTANCE_SMTP_RELAY_HOST='smtp.myhost.com',
+        INSTANCE_SMTP_RELAY_PORT=2525,
+        INSTANCE_SMTP_RELAY_USERNAME='smtpuser',
+        INSTANCE_SMTP_RELAY_PASSWORD='smtppass',
+        INSTANCE_SMTP_RELAY_SENDER_DOMAIN='opencraft.hosting'
+    )
+    @patch_services
+    @patch('instance.models.openedx_instance.OpenEdXAppServer.provision', return_value=True)
+    def test_postfix_queue_configuration(self, mocks, mock_provision):
+        """
+        Check that ansible vars for postfix_queue role are set correctly
+        """
+        instance = OpenEdXInstanceFactory(
+            sub_domain='test.postfix', use_ephemeral_databases=True, email='test.postfix@myinstance.org',
+            external_lms_domain='lms.myinstance.org'
+        )
+        appserver_id = instance.spawn_appserver()
+        appserver = instance.appserver_set.get(pk=appserver_id)
+        configuration_vars = yaml.load(appserver.configuration_settings)
+        self.assertEqual(configuration_vars['POSTFIX_QUEUE_EXTERNAL_SMTP_HOST'], 'smtp.myhost.com')
+        self.assertEqual(configuration_vars['POSTFIX_QUEUE_EXTERNAL_SMTP_PORT'], '2525')
+        self.assertEqual(configuration_vars['POSTFIX_QUEUE_EXTERNAL_SMTP_USER'], 'smtpuser')
+        self.assertEqual(configuration_vars['POSTFIX_QUEUE_EXTERNAL_SMTP_PASSWORD'], 'smtppass')
+        self.assertEqual(configuration_vars['POSTFIX_QUEUE_HEADER_CHECKS'], '/^From:(.*)$/   PREPEND Reply-To:$1')
+        self.assertEqual(
+            configuration_vars['POSTFIX_QUEUE_SENDER_CANONICAL_MAPS'],
+            'test.postfix@myinstance.org  lms.myinstance.org@opencraft.hosting'
+        )
+
```

Comment:
@mtyaka I think you could put this test in `test_openedx_appserver.py` (since the code above treats SMTP relay settings as appserver-level settings).